### PR TITLE
Update Summarization notebook to use document ai for chunking

### PR DESCRIPTION
**Pull request by holtskinner**

### CODE REVIEW

File: language/use-cases/document-summarization/summarization_with_documentai.ipynb [hunk 1]
```diff
@@ -402,21 +401,30 @@
...
+    "    \n",
+    "    # Using Document AI Toolbox to handle post-processing\n",
+    "    wrapped_document = Document.from_documentai_document(document)\n",
+    "\n",
+    "    # Split the text into chunks of the specified size based on layout.\n",
+    "    document_chunks = [block.text for page in wrapped_document.pages for block in page.blocks]\n",
+    "\n",
+    "    # Can also split into chunks by page or paragraph.\n",
+    "    # document_chunks = [page.text for page in wrapped_document.pages]\n",
+    "    # document_chunks = [paragraph.text for page in wrapped_document.pages for paragraph in page.paragraphs]\n",
```
Comment (koverholt) [inline language/use-cases/document-summarization/summarization_with_documentai.ipynb:415]: Nice extra commented examples!

File: language/use-cases/document-summarization/summarization_with_documentai.ipynb [hunk 2]
```diff
@@ -384,12 +385,10 @@
    "source": [
     "import glob\n",
     "import os\n",
-    "import textwrap\n",
     "from typing import Dict, List\n",
     "\n",
     "# If you already have a Document AI Processor in your project, assign the full processor resource name here.\n",
     "processor_name = processor.name\n",
-    "chunk_size = 5000\n",
```
Comment (koverholt) [inline language/use-cases/document-summarization/summarization_with_documentai.ipynb:392]: I think the explanatory text above on line 371 about `we are taking 5000 as a chunk_value` is outdated now?

File: language/use-cases/document-summarization/summarization_with_documentai.ipynb [hunk 3]
```diff
@@ -402,21 +401,30 @@
...
-    "                    \"content\": chunk_content,\n",
-    "                }\n",
-    "            )"
+    "    if not document:\n",
+    "        continue\n",
+    "    \n",
+    "    # Using Document AI Toolbox to handle post-processing\n",
+    "    wrapped_document = Document.from_documentai_document(document)\n",
+    "\n",
+    "    # Split the text into chunks of the specified size based on layout.\n",
```
Comment (koverholt) [inline language/use-cases/document-summarization/summarization_with_documentai.ipynb:410]: I don't quite understand how this is chunking the content based on this comment. Is it possible to reword/simplify this comment similar to the other examples below that are "chunked by page" or "chunked by paragraph"?

**Review by koverholt — APPROVED** [review]
Hooray for more clear guidance/examples on chunking! Love it.

File: language/use-cases/document-summarization/summarization_with_documentai.ipynb (hunk 2 above)
Comment (holtskinner) [reply language/use-cases/document-summarization/summarization_with_documentai.ipynb:392]: Good catch!

File: language/use-cases/document-summarization/summarization_with_documentai.ipynb (hunk 3 above)
Comment (holtskinner) [reply language/use-cases/document-summarization/summarization_with_documentai.ipynb:410]: Updated

**Pull request merged**